src/Microsoft.DocAsCode.Build.ConceptualDocuments/ConceptualDocumentProcessor.cs

```diff
@@ -62,6 +62,17 @@ public override ProcessingPriority GetProcessingPriority(FileAndType file)
             }
             if (".md".Equals(Path.GetExtension(file.File), StringComparison.OrdinalIgnoreCase))
             {
+                var filenameWithoutExtension = Path.ChangeExtension(file.File, null);
+
+                // exclude overwrite markdown segments
+                var subExtension = Path.GetExtension(filenameWithoutExtension);
+                if ((".yml".Equals(subExtension, StringComparison.OrdinalIgnoreCase) ||
+                    ".yaml".Equals(subExtension, StringComparison.OrdinalIgnoreCase))
+                    && EnvironmentContext.FileAbstractLayer.Exists(filenameWithoutExtension))
+                {
+                    return ProcessingPriority.NotSupported;
```

Comment:
use `Low`?

Reply:
There's no dedicated processor for new overwrite, so `Low` can't work.